Documentation/18 - Working with databases and HTML.md

```diff
@@ -42,9 +42,9 @@ Once that's done, you can return HTML nodes as responses:
 
 ### Databases
 
-Meridian does not include a way to talk to databases. However, any synchronous database library will work fine. If you're using Postgres, you can use this dependency:
+Meridian does not include a way to talk to databases. However, any synchronous database library will work fine. If you're using Postgres, you can use [this dependency](https://github.com/khanlou/SwiftgresQL):
 
-    .package(name: "SwiftgreSQL", url: "https://github.com/khanlou/SwiftgresQL", from: "0.1.4"),
+    .package(url: "https://github.com/khanlou/SwiftgresQL", from: "0.1.6"),
```

Comment:
Meridian works with Vapor's postgres library now, which is probably a more reliable resource than my library, so I should update this.